include/micm/solver/rosenbrock.inl

```diff
@@ -31,11 +14,6 @@ namespace micm
     decompositions = 0;
     solves = 0;
     singular = 0;
-    total_update_state_time = std::chrono::nanoseconds::zero();
-    total_forcing_time = std::chrono::nanoseconds::zero();
-    total_jacobian_time = std::chrono::nanoseconds::zero();
-    total_linear_factor_time = std::chrono::nanoseconds::zero();
-    total_linear_solve_time = std::chrono::nanoseconds::zero();
```

Comment:
With these removed, the timing example will need to be updated. Also, the cuda interface has some functionality to collect the time. I'm not sure how that could fit in with your profiling class. Could you look into that?

Reply:
Yeah, I removed them from cuda files and will request reviews from Jian and Michael

Reply:
I removed the timing interface from the CUDA kernels so it should be fine to clean up these timers.